x/rewards/genesis.go

```diff
@@ -19,6 +19,11 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	if len(genState.RefillRewardsTS.TimeEntries) == 0 {
 		k.RefillRewardsPools(ctx, nil, nil)
 	}
+
+	for _, sub := range genState.IprpcSubscriptions {
```

Comment:
now you dont need this loop, it is done in setiprpcdata

Reply:
done. see https://github.com/lavanet/lava/commit/129211e0b7a83e40ba871f4bf3eb748d0e2e73c0